NEWS.md

```diff
@@ -10,6 +10,7 @@
 - The source button uses `cpp11::source_cpp()` on C++ files that have `[[cpp11::register]]` decorations (#10387)
 - New *Relative Line Numbers* preference for showing line numbers relative to the current line, rather than the first line (#1774)
 - Upgraded SOCI library dependency from version 4.0.0 to 4.0.3 (#10792)
+- (macOS only) RStudio now reads the PATH from the user's default shell on startup (#10551)
```

Comment:
I know we mentioned potentially considering this for Linux users; however my instinct is, we've not seen any questions/concerns/issues from Linux users about the path not being as expected, so I think there's no strong justification for extending it now -- it's easy enough to change that later if it did turn out to be a need for Linux users as well, but no reason to pre-emptively do so

Reply:
That sounds good to me as well.